packages/react/src/primitives/shared/utils.ts

```diff
@@ -1,116 +1,21 @@
 import * as React from 'react';
-import autoprefixer from 'autoprefixer';
-import postcssJs from 'postcss-js';
 
 // Note: this makes nanoid more performant, not less secure
 // @see https://www.npmjs.com/package/nanoid#user-content-non-secure
 import { customAlphabet } from 'nanoid/non-secure';
 const nanoid = customAlphabet('1234567890abcdef', 12);
 
-import {
-  ComponentPropsToStylePropsMap,
-  ComponentPropToStyleProp,
-  ViewProps,
-} from '../types/index';
-
-import { getValueAtCurrentBreakpoint } from './responsive/utils';
-import { useBreakpoint } from './responsive/useBreakpoint';
-import { Breakpoint, Breakpoints } from '../types/responsive';
-
-import { useTheming } from '../../theming/';
-
-export const prefixer = postcssJs.sync([autoprefixer]);
-
 export const strHasLength = (str: unknown): str is string =>
   typeof str === 'string' && str.length > 0;
 
-export const usePropStyles = (props: ViewProps, style: any) => {
-  const {
-    theme: {
-      breakpoints: {
-        values: breakpoints,
-        unit: breakpointUnit,
-        defaultBreakpoint,
-      },
-    },
-  } = useTheming();
+export const isFn = (fn: unknown): fn is Function => typeof fn === 'function';
```

Comment:
minor: can we call it `isFunction`? Also, I couldn't find any reference to this typeguard in our codebase. Why do we need this?

Reply:
Yes, will update the name. We are using it in some new PR's that haven't been merged yet.